### DISCUSSION TITLE
tech/word/page-number-different/

### DISCUSSION THREAD

**Discussion by giscus** [tedliou/blog-discussion#1] [General]
# software/word-diff-page-number/

學術論文的封面、目錄和內文頁碼需要套用不同的設定，在 Word 中得使用分節符號來做切割。本文將說明如何為內容進行分節，讓頁碼能分別套用不同格式。

https://tedliou.com/software/word-diff-page-number/

**Comment by tedliou** [comment]
新的留言板測試~
這個沒有廣告，只是需要你登入 GitHub 帳號，快去註冊一個吧！
